app/forms/idv/api_image_upload_form.rb

```diff
@@ -235,10 +235,10 @@ def image_metadata
       @image_metadata ||= params.permit(:front_image_metadata, :back_image_metadata).
         to_h.
         transform_values do |str|
-          JSON.parse(str)
-        rescue JSON::ParserError
-          nil
-        end.
+                            JSON.parse(str)
+      rescue JSON::ParserError
+        nil
+                          end.
```

Comment:
But what in the world is this? This looks like it's messing up the block. Why is `JSON.parse(str)` so indented??